### DISCUSSION TITLE
Will the "New Authenticated Attachment URL Parameters" break my existing Discord channel archives?

### DISCUSSION THREAD

**Discussion by lera2od** [Tyrrrz/DiscordChatExporter#1136] [Q&A]
Discord announced the new "New Authenticated Attachment URL Parameters" system in the discord developers discord server(https://discord.com/channels/613425648685547541/697138785317814292/1157372186160537750). Is this gonna effect my old channel archives?

**Comment by Tyrrrz** [comment]
Probably not immediately but eventually likely so
